README.zh.md

```diff
@@ -0,0 +1,355 @@
+- [FLARE (flare-zh-corpus)](https://huggingface.co/datasets/ChanceFocus/flare-zh-corpus)
+- [FLARE (flare-zh-fineval)](https://huggingface.co/datasets/ChanceFocus/flare-zh-fineval)
+- [FLARE (flare-zh-fe)](https://huggingface.co/datasets/ChanceFocus/flare-zh-fe)
+- [FLARE (flare-zh-nl)](https://huggingface.co/datasets/ChanceFocus/flare-zh-nl)
+- [FLARE (flare-zh-nl2)](https://huggingface.co/datasets/ChanceFocus/flare-zh-nl2)
+- [FLARE (flare-zh-nsp)](https://huggingface.co/datasets/ChanceFocus/flare-zh-nsp)
+- [FLARE (flare-zh-re)](https://huggingface.co/datasets/ChanceFocus/flare-zh-re)
+- [FLARE (flare-zh-stockb)](https://huggingface.co/datasets/ChanceFocus/flare-zh-stockb)
+
+
+## Overview
+
+**FLARE_ZH** is a cornerstone initiative focusing on the Chinese financial domain, FLARE_ZH aims to bolster the progress, refinement, and assessment of Large Language Models (LLMs) tailored specifically for Chinese financial contexts. As a vital segment of the broader PIXIU endeavor, FLARE_ZH stands as a testament to the commitment in harnessing the capabilities of LLMs, ensuring that financial professionals and enthusiasts in the Chinese-speaking world have top-tier linguistic tools at their disposal.
+
+### Key Features
+
+- **Open resources**: PIXIU openly provides the financial LLM, instruction tuning data, and datasets included in the evaluation benchmark to encourage open research and transparency.
+- **Multi-task**: The instruction tuning data and benchmark in PIXIU cover a diverse set of financial tasks.
+- **Multi-modality**: PIXIU's instruction tuning data and benchmark consist of multi-modality financial data, including time series data from the stock movement prediction task. It covers various types of financial texts, including reports, news articles, tweets, and regulatory filings.
+- **Diversity**: Unlike previous benchmarks focusing mainly on financial NLP tasks, PIXIU's evaluation benchmark includes critical financial prediction tasks aligned with real-world scenarios, making it more challenging.
+
+---
+
+## FLARE_ZH: Financial Language Understanding and Prediction Evaluation Benchmark
+
+In this section, we provide a detailed performance analysis of FinMA compared to other leading models, including ChatGPT, GPT-4, lince-zero et al. For this analysis, we've chosen a range of tasks and metrics that span various aspects of financial Natural Language Processing and financial prediction.
+
+### Tasks
+
+| Data                  | Task                            | Raw     | Data Types                       | Modalities        | License           | Paper |
+| --------------------- | ------------------------------- | ------- | -------------------------------- | ----------------- | ----------------- | ----- |
+| AFQMC                 | semantic matching               | 42,511  | question data, chat              | text              | Apache-2.0        |       |
```

Comment:
Why there is no reference for AFQMC and stockB?

Reply:
> Why there is no reference for AFQMC and stockB?

There is no corresponding paper for these two data

Reply:
Just list their website if any